pyo3-macros-backend/src/pyclass.rs

```diff
@@ -615,6 +685,7 @@ fn impl_enum_class(
         methods_type,
         enum_default_methods(cls, variants.iter().map(|v| (v.ident, v.python_name()))),
         default_slots,
+        Some(impl_into_py_enum(cls, &variants)),
```

Comment:
We're changing the `IntoPy` implementation of enums. Should we maybe put this behind some flag on the `pyclass` level? Or is adhering to the python implementation of enums sufficient to change this without such control?